### PR TITLE
Show hint that you cannot manage content with Katello when using Foreman on Debian

### PR DESCRIPTION
**Pull request by maximiliankolb**
As discussed on Monday: Improve https://docs.theforeman.org/3.7/Managing_Content/index-foreman-deb.html

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13)

### CODE REVIEW

File: guides/common/modules/snip_no-content-management-for-foreman-on-deb.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Foreman on Debian/Ubuntu does to support the Katello plug-in required to manage content.
```
Comment (ekohl) [inline guides/common/modules/snip_no-content-management-for-foreman-on-deb.adoc:1]: Isn't it the other way around?
```suggestion
The Katello plug-in to Foreman is not supported on Debian/Ubuntu.
```

File: guides/doc-Managing_Content/master.adoc [hunk 2]
```diff
@@ -68,7 +65,7 @@ include::common/assembly_importing-kickstart-repositories.adoc[leveloffset=+1]
 endif::[]
 
 endif::[]
```
Comment (ekohl) [inline guides/doc-Managing_Content/master.adoc:67]: Should this `endif:[]` be moved all the way to the bottom instead? Now it still renders on `foreman-el`.

File: guides/doc-Managing_Content/master.adoc [hunk 3]
```diff
@@ -4,17 +4,14 @@ include::common/header.adoc[]
 
 = {ContentManagementDocTitle}
 
+ifdef::foreman-deb[]
```
Comment (ekohl) [inline guides/doc-Managing_Content/master.adoc:7]: We previously had a warning for `foreman-el`, but this removes that.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: guides/common/modules/snip_no-content-management-for-foreman-on-deb.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/snip_no-content-management-for-foreman-on-deb.adoc]: Slightly reworded and applied.

File: guides/doc-Managing_Content/master.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/doc-Managing_Content/master.adoc:7]: At first I though it's maybe not necessary. I've readded it & also simplified the whole "master.adoc" file. Please re-review.

File: guides/doc-Managing_Content/master.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/doc-Managing_Content/master.adoc]: Fixed.

**Review by maximiliankolb — COMMENTED** [review]
Reworked. Please re-review.

**Review by ekohl — COMMENTED** [review]
One slight nuance, but otherwise :+1: if the ISS changes to Orcharhino are intended.

File: guides/doc-Managing_Content/master.adoc [hunk 4]
```diff
@@ -4,24 +4,26 @@ include::common/header.adoc[]
...
+The Katello plug-in required to manage content in Foreman is not supported on Debian/Ubuntu.
+endif::[]
+
+ifdef::foreman-el,foreman-deb[]
+If you want to manage content using Foreman/Katello, you have to run Foreman/Katello on {EL}.
+You cannot convert an existing Foreman installation on {EL} to Foreman/Katello.
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/doc-Managing_Content/master.adoc:13]: Applied.

**Review by ekohl — APPROVED** [review]
https://github.com/theforeman/foreman-documentation/pull/2400#issuecomment-1697626080 does show a non-trivial diff for orcharhino, but I trust you know what you're doing there.

**Comment by maximiliankolb** [comment]
> [#2400 (comment)](https://github.com/theforeman/foreman-documentation/pull/2400#issuecomment-1697626080) does show a non-trivial diff for orcharhino, but I trust you know what you're doing there.

Including the appendix about kickstart file repos was by design.

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   d6860a8b5..beaa0fcf9  3.8 -> 3.8
   2e77d309b..add6ab200  3.7 -> 3.7
   88dd9dfa3..c5889350a  3.6 -> 3.6
   148eade7a..fd105c939  3.5 -> 3.5

**Pull request merged**